pyfun_events/run.py

```diff
@@ -1,45 +1,47 @@
 #!/usr/bin/python3
 #
-# A simple function to dump delivered events to stdout and report them
-# via web page.
-
-import logging
-from glob import glob
-import importlib
-from inspect import signature, isfunction
-import os.path
+# A simple function framework
 
 from cloudevents.sdk.event import v02
 from cloudevents.sdk import marshaller
 from flask import Flask, request
-from typing import Callable, TypeVar, Mapping
+from typing import Any, Callable, Optional, TypeVar, Union, overload, cast
 import ujson
 
-import pyfun_events.run
+app = Flask(__name__)
+
+T = TypeVar('T')
 
+Handler = Callable[[Any, dict], None]
 
-app = Flask(__name__)
+GetHandler = Callable[[], None]
 
+@overload
+def Handle(path: Handler) -> Handler:
+    ...
 
-def Handle(unpack: Callable[[str], object]=ujson.loads, path: str='/', **kwargs):
+@overload
+def Handle(
+      path: str="/",
+      unpack: Callable[[str], T]=ujson.loads,
+      **kwargs) -> Callable[[Callable[[T, dict], None]], Callable[[T, dict], None]]:
```

Comment:
Thanks! I couldn't figure out how to spell this, and I feel much less bad about that reading how it's done.